include/arch/arm/arch/model/statedata.h

```diff
@@ -17,6 +17,10 @@
 #include <arch/machine/debug_conf.h>
 #include <mode/machine/registerset.h>
 
+#ifdef CONFIG_ARM_HYPERVISOR_SUPPORT
```

Comment:
since you raise this, this include has already been added (https://github.com/seL4/seL4/blob/master/include/arch/arm/arch/model/statedata.h#L15)

Reply:
ah cool!